dae/dae/import_tools/import_tools.py

```diff
@@ -122,10 +122,12 @@ def get_pedigree_params(self) -> Tuple[str, Dict[str, Any]]:
         """Get params for loading the pedigree."""
         families_filename = self.get_pedigree_filename()
 
+        default_families_params = FamiliesLoader.cli_defaults()
         families_params = self.import_config["input"]["pedigree"]
         families_params = self._add_loader_prefix(families_params, "ped_")
+        default_families_params.update(families_params)
 
-        return families_filename, families_params
+        return families_filename, default_families_params
```

Comment:
The `default_families_params` name for the returned value is confusing. At least, I was confused.